Documentation/README_Slate.md

```diff
@@ -14,15 +14,13 @@ A slate control is composed of the following elements:
 * **BackPlate**: The back side of the slate.
 * **ContentQuad**: Content is assigned as material. The example uses a sample material 'PanContent'.
 
-<img src="../Documentation/Images/Slate/MRTK_Slate_Structure.png" width="650">
+<img src="../Documentation/Images/Slate/MRTK_SlateStructure.jpg" width="650">
```

Comment:
Oh interesting, any reason why we use png for most of our images, but jpg for a few here?

Reply:
I started converting some of the images to JPG if it reduces file size a lot. For the images that do not show much difference, I just left them as PNG.